ui/apps/platform/cypress/helpers/networkGraph.js

```diff
@@ -134,12 +134,45 @@ export function selectDeploymentFilter(deploymentName) {
     cy.wait(['@networkGraph', '@networkPolicies']);
 }
 
-export function selectNamespaceFilters(...namespaces) {
+// Additional calls in a test can select additional namespaces.
+
+export function selectNamespaceFilter(namespace) {
```

Comment:
A nice benefit of doing things this way is that we can still select multiple namespaces with an additional function call and be sure that the changes and requests complete serially instead of in arbitrary order.